database/gdb/gdb_model_update.go

```diff
@@ -73,7 +73,7 @@ func (m *Model) Update(dataAndWhere ...interface{}) (result sql.Result, err erro
 		// Automatically update the record updating time.
 		if fieldNameUpdate != "" {
 			if fieldNameUpdate != "" && !gstr.Contains(updates, fieldNameUpdate) {
-				updates += fmt.Sprintf(`,%s='%s'`, fieldNameUpdate, gtime.Now().String())
+				updates += fmt.Sprintf(`,%s='%s'`, fieldNameUpdate, gtime.Now().Local().Time.Format("2006-01-02 15:04:05.999999"))
```

Comment:
Is it better using time object like `gtime.Now()` instead of time string？like:
```go
updates += fmt.Sprintf(`,%s=?`, fieldNameUpdate)
conditionArgs = append(conditionArgs, gtime.Now())
```

Reply:
Yeah. this solution eliminates the conversion from time to string.